DIY Reminders with Azure and Twilio/blog.md

```diff
@@ -0,0 +1,74 @@
+
+### Step 2: Setting up Twilio Credentials
+​	Firstly, let's make sure we have Twilio installed. We'll go to the left panel of our app menu in Azure, and select console. Then type: 
+
+​	npm install Twilio
+
+<img src=".\pics\05.PNG" style="zoom:75%;" />
+
+​	Now let's go over to our Twilio account and purchase a phone number. The free Twilio trial comes with some balance so we can purchase a phone number and send calls and texts using that balance.
+
+​	Back over in Azure, let's go to the same left panel as earlier and go into the Configuration tab. There let's create 4 new application settings for your TWILIO_SID, TWILIO_TOKEN, SENDER_NUMBER, RECIPIENT_NUMBER. Those are all values that we need in our code later, but for security, we put those values under application settings instead of directly writing it into our code. 
+
+<img src=".\pics\06.PNG" style="zoom:75%;" />
+
+### Step 3: Sending a SMS
+
+​	And now, the code. Let's go back to that left menu, select "Functions" and click on the timer trigger we made earlier. There will be a "Code + Test" and we'll be editing index.js.
+
+​	Here's a look at the code:
+
+<img src=".\pics\08.PNG" style="zoom:75%;" />
+
+​	It's not too complex. If the timer is past due, write that in the log, then create and send a message from SENDER_NUMBER to RECIPIENT_NUMBER. If everything works perfectly, log a success, else log the error. 
+
+### Step 4: Making a Call
+
+​	The code to make a call isn't that different from sending a text, but we will have to make some changes. 
+
+​	One of the things that we'll have to do is make a TwiML Bin in Twilio. 
+
+<img src=".\pics\09.PNG" style="zoom:100%;" />
+
+<img src=".\pics\10.PNG" style="zoom:75%;" />
+
+​	In that TwiML Bin, let's write some code so the program knows what to say during the call, and copy the URL for the bin and save it as an Application Setting with all our other Twilio credentials. 
+
+<img src=".\pics\12.PNG" style="zoom:75%;" />
+
+​	Now we can go back and change the code like so:
+
+<img src=".\pics\11.PNG" style="zoom:75%;" />
+
+###  Step 5: Testing and We're Done!
+
+​	With this, we're all done with the code and should test it. Just click the "test" button up top and when the time you set on the timer rolls around it will go off and either send a text or call depending on which one you chose to implement. 
+
+<img src=".\pics\13.PNG" style="zoom:75%;" />
+
+​	If everything works, we're done! We've made an automated text or call that is sent at the time we set to remind us to do something we need to using Azure, Twilio, and a bit of JavaScript. You can customize the code to do whatever you want. You can use it as a wakeup call if you want something more than a simple alarm, or spam a friend with texts (probably shouldn't do that one...). Have fun with it! The sky's the limit. Now if you'll excuse me, I'm going to hit the trails.
```

Comment:
Is there a prompt for a successful test you can show?

Reply:
there is no output prompt
it just calls or texts the "recipient number"